packages/dotcom-ui-header/src/components/sticky/partials.tsx

```diff
@@ -132,31 +106,8 @@ const TopColumnCenterSticky = (props: THeaderProps) => {
   )
 }
 
-const NavListRightLoggedInSticky = (props: THeaderProps) => {
-  const subscribeAction = props.data['navbar-right-anon'].items?.[1]
-  return (
-    <React.Fragment>
-      {!props.userIsSubscribed && subscribeAction && (
-        <SubscribeButton item={subscribeAction} variant="sticky" className="o-header__top-button--hide-m" />
-      )}
-      <MyFtSticky className="" />
-    </React.Fragment>
-  )
-}
-
-// This behaviour is similar to `NavListRight` in '../navigation/partials' but:
-// - The sticky header renders either the `navbar-right-anon` data or the myFT component
-// - The normal header renders either the `navbar-right-anon` or the `navbar-right` data
 const TopColumnRightSticky = (props: THeaderProps) => {
-  let children: JSX.Element | undefined = undefined
-
-  if (props.userIsLoggedIn) {
-    children = <NavListRightLoggedInSticky {...props} />
-  } else if (props.showUserNavigation) {
-    children = <NavListRightAnonSticky {...props} />
-  }
-
-  return <div className="o-header__top-column o-header__top-column--right">{children}</div>
+  return <TopColumnRight {...props} />
```

Comment:
**Reviewer note:** Theoretically this `TopColumnRightSticky` component is no longer needed as the place it is used from could just use `TopColumnRight` directly. However in the place that `TopColumnRightSticky` is used ([here](https://github.com/Financial-Times/dotcom-page-kit/pull/1042/files#diff-66f378e331d22f88bd01c75c962869f3fe338ebcb2ae2a1f2482762522439989R71-R79)) it is building up a component out of `sticky` components, so I felt it kept a better consistency to keep this component as a proxy to `TopColumnRight`

Reply:
I feel like I have come across this before. I agree it is best to keep this as a proxy for now.